sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs

```diff
@@ -0,0 +1,270 @@
+                            new PerpetualProperty($"Order/{testScope}/Account", new PropertyValue("J Wilson")),
+                            new PerpetualProperty($"Order/{testScope}/Strategy", new PropertyValue("RiskArb")),
+                        },
+                        lusidInstrument: "CCY_ZZZ" 
+                    )
+                });
+
+            // We can ask the Orders API to upsert this order for us
+            var upsertResult = _ordersApi.UpsertOrders(scope: testScope, request: request);
+
+            // The return gives us a list of orders upserted, and the lusidinstrument for each has been mapped from the
+            // instrument identifiers passed
+            Assert.That(upsertResult.Values.Count(rl => rl.Id.Id.Equals(orderId)), Is.EqualTo(1));
+            Assert.That(upsertResult.Values.Single(rl => rl.Id.Id.Equals(orderId)).LusidInstrument, Is.EqualTo(_instrumentIds.First()));
+        }
+        
+        [Test]
+        public void Upsert_Simple_Order_With_Unknown_Instrument()
+        {
+            var testScope = $"TestScope-{Guid.NewGuid().ToString()}";
+            var orderId = $"Order-{Guid.NewGuid().ToString()}";
+            
+            // We want to make a request for a single order. We'll map the internal security id to an Unknown placeholder
+            // if we can't translate it.
+            var request = new OrderSetRequest(
+                orderRequests: new List<OrderRequest>
+                {
+                    new OrderRequest(
+                        code: orderId,
+                        quantity: 100,
+                        instrumentIdentifiers: new Dictionary<string, string>()
+                        {
+                            ["Instrument/default/LusidInstrumentId"] = "LUID_SomeNonexistentInstrument"
+                        },
+                        portfolio: new ResourceId(testScope, "OrdersTestPortfolio"),
+                        properties: new List<PerpetualProperty>
+                        {
+                            new PerpetualProperty($"Order/{testScope}/TIF", new PropertyValue("GTC")),
+                            new PerpetualProperty($"Order/{testScope}/OrderBook", new PropertyValue("UK Test Orders")),
+                            new PerpetualProperty($"Order/{testScope}/PortfolioManager", new PropertyValue("F Bar")),
+                            new PerpetualProperty($"Order/{testScope}/Account", new PropertyValue("J Wilson")),
+                            new PerpetualProperty($"Order/{testScope}/Strategy", new PropertyValue("RiskArb")),
+                        },
+                        lusidInstrument: "" 
+                    )
+                });
+
+            // We can ask the Orders API to upsert this order for us
+            var upsertResult = _ordersApi.UpsertOrders(scope: testScope, request: request);
+
+            // The return gives us a list of orders upserted, and the lusidinstrument for each has been mapped from the
```

Comment:
Does this return an Instrument or LusidInstrumentId?

Reply:
It returns a list of orders, each of ehich will contain the instrument identifiers passed in (note, not the full possible resolved set) and the newly resolved LusidInstrumentId

Reply:
perhaps update the comment from `lusidinstrument` to `LusidInstrumentId` or `LUID` so its clear

Reply:
Done